UI improvements after merge

0nko opened the pull request:
This PR adds some UI improvements in the View all screen that surfaced after a significant merge from `develop`, namely:
- Adds support for the offline message snackbar
- Makes the loading indicator style consistent with the main stats screen
- Extracts the tabs of a particular use case outside the card, so that it's pinned and scrolled with the toolbar
- Fixes the loading of more data in the View all followers screen
- Fixes the unit tests

0nko committed:
Display an error message in a snackbar if problem loading data

0nko committed:
Add the R qualifier for better navigation

0nko committed:
Use the fragment tabs with scrolling if a use case uses tab items

0nko committed:
Remove the unnecessary error and empty layout

0nko committed:
Fix the test build

0nko committed:
Fix loading more followers

0nko committed:
Display the View more/Load more button only when the relevant tab is selected

0nko committed:
Fix a failing unit test

0nko committed:
Fix ktlint

0nko committed:
Remove obsolete coroutines setting

planarvoid commented:
Results from testing:
> Adds support for the offline message snackbar
works nicely 👍, should we add an offline message on `Load more`? maybe in another PR 
> Makes the loading indicator style consistent with the main stats screen
looks great 👍but it only shows on the View all Followers screen
> Extracts the tabs of a particular use case outside the card, so that it's pinned and scrolled with the toolbar
Looks good, there is a bug in scrolling app bar layout when on a screen without tabs - as discussed in DM
* Fixes the loading of more data in the View all followers screen
> Works well 👍

0nko committed:
Disable the app bar scrolling by default

0nko committed:
Open currently selected card Followers tab when tapping the View all button

0nko committed:
Open currently selected card Comments tab when tapping the View all button

0nko commented:
> should we add an offline message on `Load more`?

It shows the message the first time. It would require deeper snackbar refactoring so let's leave it for a different PR.

> Makes the loading indicator style consistent with the main stats screen

The loading seems to take a lot shorter for other use cases somehow so it's not easy to notice, it does show it, though.

> Looks good, there is a bug in scrolling app bar layout when on a screen without tabs

This is now fixed.

I also implemented currently selected tab opening - the same tab that's selected on the stats card is opened in the View all screen.

Additionally, I added a check that now shows the View more button in Stats and Load more in View all screen only for a tab that has more data to show.

0nko committed:
Fix ktlint

0nko committed:
Optimize imports

0nko committed:
Fix failing unit tests

0nko committed:
Merge branch 'issue/8815-view-more-screen' into feature/view-all-improvements

0nko committed:
Update FluxC reference

0nko committed:
Fix the FluxC reference

planarvoid reviewed (approved):
looks good 👍  :shipit:
